File: BACKEND/ini.js
var mqtt = require('mqtt');
var mqttClient = mqtt.connect("mqtt://user1:cr7@m13.cloudmqtt.com:18654");
mqttClient.on('connect', function () {
  console.log("MQTT connected.")
  // TODO subscribe to the topic
});
 
mqttClient.on('message', function (topic, message) {
  console.log("Topic: " + topic);
  console.log("Message: " + message.toString());
});

File: BACKEND/ini.py
import paho.mqtt.client as mqtt
import urlparse

# Init MQTT client
mqttClient = mqtt.Client()
# e.g. mqtt://username:password@m20.cloudmqtt.com:port
mqttConnectionString = "mqtt://user1:cr7@m13.cloudmqtt.com:18654"
url = urlparse.urlparse(mqttConnectionString) 	
mqttClient.username_pw_set(url.username, url.password)
mqttClient.connect(url.hostname, url.port)


